features/cellular/easy_cellular/EasyCellularConnection.cpp

```diff
@@ -42,7 +42,7 @@ bool EasyCellularConnection::cellular_status(int state, int next_state)
 
     if (_target_state == state) {
         tr_info("Target state reached: %s", _cellularConnectionFSM->get_state_string(_target_state));
-        MBED_ASSERT(_cellularSemaphore.release() == osOK);
+        (void)_cellularSemaphore.release();
```

Comment:
Looks better, but we ignore the return value?

Reply:
Yes, we ignore it. There is nothing that can be done and no point doing if's just for printing semaphore release error which should never happen.

Reply:
@AnttiKauppila Quick thoughts on the above response?